Lib/xml/sax/expatreader.py

```diff
@@ -91,6 +91,7 @@ def __init__(self, namespaceHandling=0, bufsize=2**16-20):
         self._entity_stack = []
         self._external_ges = 0
         self._interning = None
+        self._namespace_prefixes = 1
```

Comment:
Why is it 1 by default?

Reply:
Changed to use a new `namespacePrefixesHandling` argument.